style.css

```diff
@@ -420,3 +420,103 @@ ul {
   color: #e2505c;
   border: 1px solid #e2505c;
 }
+
+.form-section {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  grid-gap: 24px;
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 288px;
+}
+
+form > input {
+  height: 52px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+  padding-left: 16px;
+  border-radius: 8px;
+}
+
+.form-section > p {
+  width: 288px;
+  height: 63px;
+  left: 16px;
+  margin-top: 1px;
+}
+
+.form-section > h1 {
+  width: 350px;
+  height: 50px;
+  left: 16px;
+  font-family: "Playfair Display", sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  text-align: left;
+  padding: 1px;
+  margin-top: 106px;
+}
+
+#cbn-b {
+  width: 288px;
+  height: 44px;
+  left: 16px;
+  top: 4652px;
+  background: #1a2236;
+  color: #fff;
+}
```

Comment:
- [ ] Why don't you give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray: